### DISCUSSION TITLE
Anchors and quest 2

### DISCUSSION THREAD

**Discussion by mvvvv** [StereoKit/StereoKit#878] [Q&A]
Hi! I wrote the anchor demos for rust : <https://github.com/mvvvv/StereoKit-rust/blob/master/examples/demos/anchor1.rs>

This works perfectly in the simulator under Linux but not under Quest 2 where the Anchor.Capabilities are systematically None. I tried different settings (which relate more to OVR than OpenXR). Is there a way to make it work under Quest?

[log.txt](https://github.com/StereoKit/StereoKit/files/14437029/log.txt)

**Comment by maluoi** [comment]
I mentioned in the release notes, but unfortunately I didn't manage to fit the FB anchors in v0.3.9! Just the MSFT ones. I've got some [WIP code](https://github.com/StereoKit/StereoKit/tree/feature/anchors) for FB anchors, but it's not functional yet.

If I finish it before v0.4, I may try and put together a v0.3.10 with just FB anchors in there, but right now I'm working on some other high-priority features that have kept me pretty busy.
